src/multi_optimization_container.jl

```diff
@@ -42,28 +42,28 @@ function MultiOptimizationContainer(
     )
 
     return MultiOptimizationContainer{T}(;
-        main_problem = PSI.OptimizationContainer(sys, settings, nothing, U),
```

Comment:
You must be using the wrong formatter settings.

Reply:
I am running the formatter in the scripts folder in the repo.